R/get_v1_data.R

```diff
@@ -273,28 +290,30 @@ spod_clean_zones_v1 <- function(zones_path) {
 #' # create a connection to the v1 data
 #' Sys.setenv(SPANISH_OD_DATA_DIR = "~/home/nosync/cache/mitma")
 #' dates <- c("2020-02-14", "2020-03-14", "2021-02-14", "2021-02-14", "2021-02-15")
-#' od_dist <- spod_get_od_v1(zones = "distr", dates = dates)
+#' od_dist <- spod_get_od(zones = "distr", dates = dates)
 #'
 #' # od dist is a table view filtered to the specified dates
 #'
 #' # access the source connection with all dates
 #' # list tables
 #' DBI::dbListTables(od_dist$src$con)
 #' }
-spod_get_od_v1 <- function(
+spod_get_od <- function(
     zones = c(
       "districts", "dist", "distr", "distritos",
       "municipalities", "muni", "municip", "municipios"
     ),
     dates = NULL,
     data_dir = spod_get_data_dir(),
     quiet = FALSE,
-    duck_max_mem = 2,
-    duck_max_threads = parallelly::availableCores())
+    duck_max_mem = 3,
+    duck_max_threads = parallelly::availableCores() - 1)
```

Comment:
👍 to not max out machines!

Reply:
yep, to keep one core to prevent the OS from freezing